src/EnergyPlus/Autosizing/CoolingCapacitySizing.cc

```diff
@@ -551,7 +551,7 @@ Real64 CoolingCapacitySizer::size(EnergyPlusData &state, Real64 _originalValue,
         if (this->isEpJSON) this->sizingString = "cooling_design_capacity [W]";
     }
     if (this->dataScalableCapSizingON) {
-        auto const SELECT_CASE_var(this->zoneEqSizing(this->curZoneEqNum).SizingMethod(DataHVACGlobals::CoolingCapacitySizing));
+        auto const &SELECT_CASE_var(this->zoneEqSizing(this->curZoneEqNum).SizingMethod(DataHVACGlobals::CoolingCapacitySizing));
```

Comment:
Right now this is constexpr so this should not use & ? And if it were changed to enum it should also not use & ?

    constexpr int CapacityPerFloorArea(10);

Reply:
Variables cannot `constexpr` because you can assign different values to them. The fact that `CapacityPerFloorArea` is `constexpr` does not make `this->zoneEqSizing(this->curZoneEqNum).SizingMethod(DataHVACGlobals::CoolingCapacitySizing)` itself `constexpr`. And whether a variable is assigned to a value that is `constexpr` or not doesn't matter here. What matters is the type of the variable, which is `int`. So, two things here.

First, when creating a local shortcut to a scalar (like an `int`) do a copy by value. Adding a reference doesn't buy you anything. Here is an [explainer](https://github.com/NREL/EnergyPlus/wiki/Local-Shortcut-Variables%3A-When-to-Use-a-Value-Copy-and-When-to-Use-a-Reference) on this.    

Second, use `auto` only for (non-standard library) object references and iterators. `int` is not one of those. In this case, just use `int`.